docs/pages/versions/unversioned/sdk/payments.md

```diff
@@ -1,771 +0,0 @@
----
```

Comment:
Shouldn't we keep this page for a while? It may still appear in search results or sth, so here maybe we can redirect people to use stripe-react-native

Reply:
the older sdk version pages will still be around

Reply:
![Group 1](https://user-images.githubusercontent.com/90494/132906595-b3efe9d8-1e41-4b0d-a33e-ba4dcefa092e.png)